src/databricks/labs/remorph/cli.py

```diff
@@ -96,6 +96,21 @@ def _get_spark_session(ws: WorkspaceClient) -> SparkSession:
     return DatabricksSession.builder.sdkConfig(ws.config).getOrCreate()
 
 
+@remorph.command
+def generate_recon_config(w: WorkspaceClient):
+    """generates recon_config file for reconciliation"""
+    logger.info(f"User: {w.current_user.me()}")
+    logger.info("Generating recon_config file for reconcile")
+
+    # Create Installation object
+    # installation = Installation(w, "remorph")
+    folder_path = f"/Users/{w.current_user.me().user_name}/.remorph_demo"
+    installation = Installation(w, "remorph", install_folder=folder_path)
```

Comment:
```suggestion
    installation = Installation.assume_user_home(ws, "remorph")
```

https://github.com/databrickslabs/blueprint?tab=readme-ov-file#detecting-current-installation

Reply:
Done